### DISCUSSION TITLE
Discover subnets from devices

### DISCUSSION THREAD

**Discussion by pahilko** [netbox-community/netbox#13755] [Q&A]
I have multivendor network infrastructure(cisco, checkpoint, vyos, etc).
And I need to track all IP subnets(vlans) on my devices.
Is it possible to automatically collect ip subnets configured on interfaces(if I add/delete subnet on my network device, it should automatiacally be added/deleted in netbox) What is the best way to do it?

**Comment by deku-m** [comment]
Maybe this plugin is what you are searching but no support for checkpoint vyos i think:
https://github.com/dainok/netdoc
